src/main/java/nl/tudelft/semgroup4/resources/ResourcesWrapper.java

```diff
@@ -367,4 +367,12 @@ public Image createImage(int width, int height) throws SlickException {
     public TrueTypeFont createFont(Font font, boolean bool) {
         return new TrueTypeFont(font, bool);
     }
+
+    public boolean getMusicOn() {
```

Comment:
*Auto-generated comment*

    
    A getX() method which returns a boolean should be named isX()